tests/unit/test_platform.cpp

```diff
@@ -302,7 +302,7 @@ otRadioFrame *otPlatRadioGetTransmitBuffer(otInstance *aInstance)
     }
     else
     {
-        return (otRadioFrame *)0;
+        return (otRadioFrame *)nullptr;
```

Comment:
```suggestion
        return nullptr;
```
nit: is the type conversion necessary?